.github/workflows/lint-test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir.outputs.YARN_CACHE_DIR }}
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
-      - run: yarn --frozen-lockfile
+      - run: yarn --immutable
       - run: yarn allow-scripts
```

Comment:
Sorry, 3 lines I guess, I missed this one
```suggestion
```

Reply:
Whoops I should've caught that